### PR TITLE
[90] Group Invitations

### PR DESCRIPTION
**Pull request by esoterik**

### CODE REVIEW

File: lib/tasks/style_checker.rake [hunk 1]
```diff
@@ -38,7 +38,7 @@ def passed?
 end
 
 def rubocop(files)
-  "rubocop -D --force-exclusion #{files}"
+  "rubocop -Da --force-exclusion #{files}"
```
Comment (esoterik) [inline lib/tasks/style_checker.rake]: need to remove

File: spec/features/groups/invite_to_join_spec.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+RSpec.feature 'Students Joining Groups' do
+  context 'requesting to join open group' do
+    it 'succeeds' do
+      group = FactoryGirl.create(:open_group)
+      user = FactoryGirl.create(:student, intent: 'on_campus', draw: group.draw)
+      log_in group.leader
+      visit draw_group_path(group.draw, group)
+      click_on 'Invite Members'
+      select user.full_name.to_s, from: 'group_member_ids'
```
Comment (orenyk) [inline spec/features/groups/invite_to_join_spec.rb]: Remove .to_s

File: app/controllers/groups_controller.rb [hunk 3]
```diff
@@ -47,6 +48,16 @@ def accept_request
     handle_action(path: draw_group_path(@draw, @group), **result)
   end
 
+  def invite_to_join
```
Comment (orenyk) [inline app/controllers/groups_controller.rb]: Change to `invite_to_user` (don't forget policy)

File: app/controllers/groups_controller.rb [hunk 4]
```diff
@@ -47,6 +48,16 @@ def accept_request
...
+    batch_params = { user_ids: params['group']['member_ids'], group: @group,
```
Comment (orenyk) [inline app/controllers/groups_controller.rb]: strong params please

File: app/services/creators/membership_batch_creator.rb [hunk 5]
```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+class MembershipBatchCreator
```
Comment (orenyk) [inline app/services/creators/membership_batch_creator.rb]: documentation for public methods

File: app/services/creators/membership_batch_creator.rb [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+  private
+
+  attr_reader :users, :successes, :failures, :results, :params
+
+  def build_result
+    @successes, @failures = results.partition { |r| r[:msg].key? :success }
+    obj = successes.first[:object] unless successes.empty?
+    {
+      object: obj || nil,
```
Comment (esoterik) [inline app/services/creators/membership_batch_creator.rb]: comment that this is not a membership

Comment (orenyk) [inline app/services/creators/membership_batch_creator.rb]: we don't need to specify nil, it comes from line 29

File: app/services/creators/membership_batch_creator.rb [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+      msg: { notice: build_flash }
```
Comment (orenyk) [inline app/services/creators/membership_batch_creator.rb]: This should have two keys, `:success` with the success messages and `:error` with the failures. The logic for generating it should be in `build_flash`.

**Pull request merged**